### DISCUSSION TITLE
Distinguish the standard from the Foundation it self

### DISCUSSION THREAD

**Discussion by janderssonse** [standard-for-public-code/standard-for-public-code#1037] [General]
As we discussed at last community call it make sense to lessen confusion for external users,  and remove some of the parts pointing directly to the Foundation. This to make clear that the standard and the Foundation is separate . I would suggest focusing on just getting an updated GitHub page for this organisation running. I think we were a few who were interested in doing this. I'll try to see if can get some work done later tonight on the github.io page regarding this.

**Comment by janderssonse** [comment]
After https://github.com/standard-for-public-code/standard-for-public-code/pull/1038 next actions
* I would suggest just continue going through the site to remove or change any implications that this is the Foundation itself.
* Get a domain name so that we can validate this GitHub organisation
* Start working on closing issues. 

Let's discuss these things on Thursday

**Comment by janderssonse** [comment]
And as reported here: https://github.com/standard-for-public-code/standard-for-public-code/issues/1024

**Comment by clausmullie** [comment]
Currently it loads in the foundation jekyll theme as a remote theme, so we could either fork and edit that theme, create a new jekyll theme, or find another theme to import. 

Some time ago I was looking at the just the docs theme - https://just-the-docs.com/ which I thought might be appropriate and suitable. It is fairly plain and simple, has a search bar built in, and gives a nice continual overview on the navigation bar.

Did a bit of work to get the just the doc theme in - you can check it out locally on this [branch](https://github.com/standard-for-public-code/standard-for-public-code/tree/sandbox-just-the-docs). It did require quite some changes to frontmatter.

**Reply by clausmullie** [reply]
I'm not super familiar with jekyll, so maybe im not aware of some magic somewhere, but something to consider is compatibility for different branding

Having a remote theme is nice, because anyone can host their own themed (and translated?) version.

But for that, it would be good to align around one way of using front matter to reduce complexity.

Eg would shifting to just the docs and changing the front matter make it harder for the foundation to pull in new changes? Building a new theme which uses front matter in the same way as the public code theme might cause less friction in this use case. (On the other hand, the foundation just did a big rebrand on the homepage so i don't know how long they will keep their current jekyll theme.) 

@mcarlson would the foundation be happy with the just the docs theme? Do you have plans to rebrand the standard page and/or the public code jekyll theme in general?

Does anyone else have a strong preference around front matter usage?

**Reply by clausmullie** [reply]
Thinking about this a bit more, it seems a shift to just the docs would include more work than anticipated.

This is because this repo and the community translations rely on the current public code remote jekyll theme, and the printing to pdf setup mimics the current theme and style.

A change to just the docs would require some structural changes to folders/files/frontmatter in multiple repos, and figuring out a new print style - so quite some work.

The easiest course of action would be to fork the public code theme, make some minor edits, and have this as a remote theme. We can then tweak the theme and the print setup in unison. 

Later on, if and when relevant, a new remote theme and accompanying print setup could be introduced.

**Reply by janderssonse** [reply]
Let's start from there (themes) as to not get overwhelmed then,  it is a good step forward ,( and thanks for the examination of the current state regarding themes and printing. )

**Comment by janderssonse** [comment]
Will try it out in a moment.:) BTW I added some agenda suggestions for the meeting today and one them is this point - further actions -. On a longer term (and there are more important actions long before that) I suggest we also  would  migrate go over to a newer jekyll for the GH.pages, the native one has not been updated in years, and they seem to propose other gh action solution. https://github.com/github/pages-gem/issues/651 - But let's get the more important stuff done first!:)

**Reply by janderssonse** [reply]
I suggest, @clausmullie Maybe you would like to demo the theme change at the meeting? Add it to the agenda if so - I added the criteria link fix as an brief demo:)

**Reply by janderssonse** [reply]
@clausmullie I would vote for this. It is simple and clean. And It would give a clear distinction. And that would give us time to "in the future, if and when we feel it is needed", just add a very custom theme.

**Comment by mcarlson** [comment]
Hello,

I think it makes sense to change the main page, e.g. we’d like the https://standard.publiccode.net/ to look more like https://publiccode.net/public-code.html. We aren’t using jekyll for the new site. This could just be for a landing page with the standard embedded in the middle - even via an iframe.

I know Ryan (cc’d) has been working on a new design, perhaps we can share that around.

Ideally we can avoid having multiple different versions/forks of the standard online by finding a solution that the community is happy with, e.g. I don’t think the foundation needs to be shown in the top-left corner anymore, now that it’s being maintained by a larger community - though the work by the foundation should be credited.

I don’t think shifting to just the docs would be an issue, either way it makes sense to change the branding. It should be whatever the community finds most productive to maintain going forward - and ideally the styling is pretty neutral and the current branding shifts to a ‘credits’ page.

Regards,
Max Carlson

**Comment by ElenaFdR** [comment]
@mcarlson, is a credits page a common way of approaching this? I've made [a suggestion](https://github.com/standard-for-public-code/standard-for-public-code/issues/1024#issuecomment-2192268165) on #1024 for a very short explanation of the community management/history, but we could also have a 1 paragraph version.
